src/tests/ftest/util/server_utils.py

```diff
@@ -39,6 +39,7 @@ class DaosServerCommand(YamlCommand):
 
     NORMAL_PATTERN = "DAOS I/O server.*started"
```

Comment:
don't like using these hardcoded strings, we should be able to query the system state through "dmg system query --verbose" to work out what's going on in a more reliable manner. MemberStateAwaitFormat has been added

Reply:
I've added this ticket for adding support in our test framework to use `dmg system query` command to address this concern: https://jira.hpdd.intel.com/browse/DAOS-4820